### PR TITLE
T42456

### PR DESCRIPTION
**Pull request by dexyarya**
# Description

Create FE-Login page  username accept number and alphabetic, username max length 12, password min length 8 create slideshow (4) items

Fixes # (issue)

## Type of change

Please select options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes and Provide evidence. ie: screenshot or screen record.
https://www.loom.com/share/11cc08f8cc874b8eb0ae916d68c4e417

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "extends": [
         "eslint:recommended",
         "plugin:react/recommended"
+        // "plugin:prettier/recommended"
```
Comment (argorfc) [inline .eslintrc.json]: Please remove unused line!

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Row, Col, Image, Select } from "antd";
+import image1 from "../../assets/logo2.png";
+import "./Header.css";
+const Logo = image1;
+const { Option } = Select;
+
+const handleChange = (value) => {
+  console.log(value);
```
Comment (argorfc) [inline src/components/header/Header.jsx]: is the value not stored ?

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            width: 136,
+          }}
+          onChange={handleChange}
+        >
+          <Option value=" English"> English (En)</Option>
+          <Option value="Indonesia">Indonesia (Ind)</Option>
+        </Select>
+      </Col>
+    </Row>
+    // </Card>
```
Comment (argorfc) [inline src/components/header/Header.jsx]: please remove unused line ?

File: src/main.jsx [hunk 4]
```diff
@@ -1,10 +1,11 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App'
-import './index.css'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App";
+import "./index.css";
```
Comment (argorfc) [inline src/main.jsx]: It must be undefined after you delete index.css, please make sure!

**Review by argorfc — CHANGES_REQUESTED** [review]
Read the comments!

**Comment by dexyarya** [comment]
already fix 55c7b566ac29119a032464a67c3bdfbd79a39326

**Pull request merged**